styles/styles.css

```diff
@@ -0,0 +1,683 @@
+        flex-direction: column;
+        /*width: 50vw;*/
+        text-align: center;
+        flex-basis: 1200px;
+        padding: 100px 115px 125px 115px;
+
+
+
+    }
+
+
+header button {
+            color: #ffffff;
+            background-color: var(--roze);
+            border-radius: 50px;
+            padding: 25px 40px;
+            cursor: pointer;
+            border: none;
+            -webkit-box-shadow: 0px 5px 5px 5px rgba(0,0,0,0.25);
+            -moz-box-shadow: 0px 5px 5px 5px rgba(0,0,0,0.25);
+            box-shadow: 0px 5px 5px 5px rgba(0,0,0,0.25);
+            width: fit-content;
+            align-self: center;
+        }
+
+.imagecontainerTopImage{
+    /*border: 2px solid turquoise;*/
+    display: flex;
+    flex-direction: row;
+    justify-content: center;
+    margin: -4vh 0 ;
+}
+
+img .topImage {
+    display: flex;
+    justify-content: center;
+    max-width: 150px;
+    max-height: 150px;
+}
+
+
+/*---------------Header end---------------*/
+/*---------------Main start---------------*/
+
+main {
+    background-color: white;
+}
+
+/*-white section in main start-*/
+.outerContainerIntroduction {
+    /*border: deeppink 2px dashed;*/
+    display: flex;
+    flex-direction: row;
+    justify-content: center;
+    margin: 0 30px;
+}
+
+.innerContainerIntroduction {
+    /*border: red 2px dashed;*/
+    display: flex;
+    flex-direction: row;
+    flex-basis: 1200px;
+    height: 88vh; /*space between white/grey*/
+    width: 40em;
+}
+
+.insideInnerContainerIntroduction {
+    /*border: deeppink 2px dashed;*/
+    display: flex;
+    flex-direction: column;
+    text-align: center;
+    padding: 120px 195px 110px 195px
+}
+
+
+
+/*-white section in main end-*/
+/*-light grey section in main start-*/
+
+main .GreyBackground {
+    background-color: var(--lichtgrijs);
+}
+
+/*-light grey section > Article Rolling stone start-*/
+
+
+.outerContainerImageRollingStoneArticle {
+    /*border: deeppink 2px dashed;*/
+    display: flex;
+    flex-direction: row;
+    justify-content: center;
+    margin: 0 30px;
+}
+
+.innerContainerImageArticleRollingStone {
+    /*border: turquoise 2px dashed;*/
+    display: flex;
+    flex-direction: row;
+    flex-grow: 1;
+    justify-content: space-between;
+}
+
+.articleRollingStoneTextFlexbox1 {
+    /*border: red 2px solid;*/
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    max-height: 600px;
+    flex-grow: 0;
+    flex-shrink: 1;
+    margin: 0 90px 0 0 ;
+}
+
+.articleRollingStoneTextFlexbox2 {
+    /*border: blue 2px solid;*/
+    display: flex;
+    flex-direction: column;
+    flex-basis: 400px;
+    padding: 1.2em 2.3em 0  3.4em ;
+    margin: 218px 0 0 10px;
+    width: 500px;
+}
+
+
+
+main section .flexImageLaptop {
+    /*border: forestgreen 2px solid;*/
+    display: flex;
+    flex-direction: column;
+    text-align: center;
+    max-height: 600px;
+    flex-grow: 0;
+    flex-shrink: 1;
+    margin: 85px 20px 0 0;
```

Comment:
Dit heeft óók allemaal geen functie. Wat nog wel handig kan zijn, is dat je de hoogte gelijkmaakt met de content:
`height: fit-content;`